### PR TITLE
Adds borer as its own gamemode. Switches some psionics around.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
- Borer has been added as its own gamemode, requiring 3 borer players and 20 people readied."
  - To be eligible as a borer host you now need the borer antag pref on. If a borer cannot find a suitable host, it will spawn in a randomly positioned landmark."
  - Zorch, the eye laser power, has been made into a lightning arc power instead. It works like before, except it shoots teslabolts."
  - Disable, the EMP ability, has been made into an emag ability. You now activate it by targeting right hand and clicking on items."
  - Increased borer psi awakening chemical cost to 150."
  - Borer psi awakening is now a bit more painful."

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/game/antagonist/alien/borer.dm [hunk 1]
```diff
@@ -42,6 +42,8 @@ var/datum/antagonist/xenos/borer/borers
 				continue
 			if(H.mind?.special_role)
 				continue
+			if(!MODE_BORER in H.client.prefs.be_special_role) //Don't draft people that don't have the borer pref on.
```
Comment (skull132) [inline code/game/antagonist/alien/borer.dm]:
```suggestion
			if(!(MODE_BORER in H.client.prefs.be_special_role)) //Don't draft people that don't have the borer pref on.
```

File: code/modules/psionics/faculties/energistics.dm [hunk 2]
```diff
@@ -7,24 +7,27 @@
 /datum/psionic_power/energistics
 	faculty = PSI_ENERGISTICS
 
-/datum/psionic_power/energistics/disrupt
-	name =            "Disrupt"
-	cost =            10
+/datum/psionic_power/energistics/sequencer
```
Comment (Arrow768) [inline code/modules/psionics/faculties/energistics.dm]: I do not believe that infinite emag is a good idea, due to the difficulty involved in undoing the damage caused by it / balance concerns

Comment (NonQueueingMatt) [reply code/modules/psionics/faculties/energistics.dm]: Alright, fair. I'll make up something to replace it.

Comment (NonQueueingMatt) [reply code/modules/psionics/faculties/energistics.dm]: This is addressed.

**Pull request merged**